app/src/main/java/comp3350/mealbuddy/objects/Macros.java

```diff
@@ -0,0 +1,18 @@
+package comp3350.mealbuddy.objects;
+
+
+import java.util.EnumMap;
+
+public class Macros {
+
+    private static EnumMap<nutrients, Integer>dictionary;
+
+    public enum nutrients{
+        OMEGA3, OMEGA6, SATURATEDFAT, PROTEIN, CARBOHYDRATES;
+    }
+
+    public static EnumMap<nutrients, Integer> makeMacros() {
+        dictionary = new EnumMap<nutrients, Integer>(nutrients.class);
+        return dictionary;
```

Comment:
the whole point of using an EnumMap, is so we can have a central dictionary for both Macro and Micro. with your implementation we will end up with two enumMaps with Macro and Micro seperate.